src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/reports/EmailSenderDisabledTest.java

```diff
@@ -1,20 +1,22 @@
 package uk.gov.hmcts.reform.sendletter.tasks.reports;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.ApplicationContext;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@SpringBootTest
-public class EmailSenderDisabledTest {
+@SpringBootTest(properties = {
+    "spring.mail.host=false"
+})
```

Comment:
how did this work before??

Reply:
It didn't. The `@Test` annotation was of junit4 and no vintage was present in test scope. For a reason - everything is (suppose to be) junit5

Reply:
What is this test for?
If it worked before without explicitly setting host in the annotation, perhaps we should also change whatever was being done before to not have report sender in context

Reply:
it did not work before. it wasn't even picked up before. non of junit4 tests were picked up

Reply:
shouldn't this host entry be moved to config file for integration tests then?

Reply:
it is moved and set to a certain dummy value. only testing the "disable" flag with these tests